chromium_src/chrome/browser/ui/views/translate/translate_bubble_view.h

```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_TRANSLATE_TRANSLATE_BUBBLE_VIEW_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_TRANSLATE_TRANSLATE_BUBBLE_VIEW_H_
+
+#define BRAVE_TRANSLATE_BUBBLE_VIEW_H_ \
+  private: \
+    friend class BraveTranslateBubbleView; \
+  public:
+class BraveTranslateBubbleView;
```

Comment:
just for readability please leave a space after the multiline macro

Reply:
addresses in https://github.com/brave/brave-core/pull/3103/commits/352251d2b14325171f11e505d7ccdcdf974caf09